integrations/docker/images/chip-build-esp32/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN set -x \
     && cd /opt/espressif \
     && git clone --progress -b release/v4.2 https://github.com/espressif/esp-idf.git \
     && cd esp-idf \
+    && git reset --hard d9ec7df3931bea46292f015a99d19a28670bd561
```

Comment:
Hey @andy31415  We need to increment the dockerfile version after this PR is merged?

Reply:
I think we can do this after merge.